src/uu/cp/src/cp.rs

```diff
@@ -2063,6 +2072,13 @@ fn print_paths(parents: bool, source: &Path, dest: &Path) {
     println!("{}", context_for(source, dest));
 }
 
+fn print_removed_path(path: &Path) {
+    println!(
+        "{}",
+        translate!("cp-verbose-removed", "path" => path.quote())
+    );
+}
+
```

Comment:
Not sure of the added value to wrap a single print in a function. I think you can inline it